app/src/main/res/values/attrs.xml

```diff
@@ -12,6 +12,8 @@
 
     <declare-styleable name="Progress">
         <attr name="text_size" format="dimension"/>
```

Comment:
Use the conventional attribute 'textSize` instead of defining your own `text_size`.